### PR TITLE
Fix yara rule filenames

### PR DESCRIPTION
**Pull request by msm-code**
The usual file extension for yara rules is `.yar`.

**Your checklist for this pull request**
- [x] I've read the [contributing guideline](https://github.com/CERT-Polska/mquery/blob/master/CONTRIBUTING.md).
- [x] I've tested my changes by building and running mquery, and testing changed functionality (if applicable)
- [x] I've added automated tests for my change (if applicable, optional)
- [n/a] I've updated documentation to reflect my change (if applicable)

**What is the current behaviour?**
Yara rules in test directory have no extension.

**What is the new behaviour?**
Yara rules in test directory have `.txt` extension.

**Test plan**
n/a

**Closing issues**

### CODE REVIEW

File: src/tests/yararules/test_corpus_yara_rule.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 
-        for file in yara_files:
-            with open(testdir + file) as f:
+        for filename in yara_files:
+            with open(testdir + filename) as f:
                 data = f.read()
 
-            expected_file_txt = file + ".txt"
+            expected_file_txt = file[:-4] + ".txt"
             self.assert_query(data, expected_file_txt)
```
Comment (ITAYC0HEN) [inline src/tests/yararules/test_corpus_yara_rule.py:24]: `file` became `filename`. and `.txt` changed to `.yar`.
What happens here?

**Review by nazywam — APPROVED** [review]
:wew:

File: src/tests/yararules/test_corpus_yara_rule.py [hunk 2]
```diff
@@ -14,13 +14,13 @@
 class TestYaraRules(unittest.TestCase):
     def test_regression(self):
 
-        yara_files = [f for f in os.listdir(testdir) if ".txt" not in f]
+        yara_files = [f for f in os.listdir(testdir) if f.endswith(".yar")]
```
Comment (nazywam) [inline src/tests/yararules/test_corpus_yara_rule.py:17]: We can actually use pathlib here

```suggestion
        yara_files = Path(testdir).glob("*.yar")
```

File: src/tests/yararules/test_corpus_yara_rule.py [hunk 3]
```diff
@@ -14,13 +14,13 @@
...
 
-        for file in yara_files:
-            with open(testdir + file) as f:
+        for filename in yara_files:
+            with open(testdir + filename) as f:
                 data = f.read()
```
Comment (nazywam) [inline src/tests/yararules/test_corpus_yara_rule.py:21]: This then becomes
```suggestion
            data = f.read_text()
```

File: src/tests/yararules/test_corpus_yara_rule.py [hunk 4]
```diff
@@ -14,13 +14,13 @@
...
 
-            expected_file_txt = file + ".txt"
+            expected_file_txt = filename[:-4] + ".txt"
```
Comment (nazywam) [inline src/tests/yararules/test_corpus_yara_rule.py:23]: And this

```suggestion
            expected_file_txt = filename.with_suffix('.txt')
```

**Review by nazywam — COMMENTED** [review]
Pathlib works pretty cool here

**Review by nazywam — APPROVED** [review]

**Comment by msm-code** [comment]
Thanks for review and suggestions @nazywam

**Pull request merged**